app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,126 @@
+content-type: explanation
+---
+
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization, both {{site.base_gateway}} services and external integration resources.
+
+<!-- vale off-->
+{% mermaid %}
+graph LR
+  %% Define styles for icons and nodes
+  classDef integration fill:#f7f7f7,stroke:#333,stroke-width:2px;
+  classDef resource fill:#e1f7d5,stroke:#333,stroke-width:2px;
+  classDef konnect fill:#d0e5ff,stroke:#333,stroke-width:2px;
+  classDef serviceCatalog fill:#e1f7d5,stroke:#333,stroke-width:2px;
+  classDef billingInventory fill:#f0f0f0,stroke:#333,stroke-width:2px;
+
+  %% Define the resources
+  GitHub[<img src="https://img.icons8.com/ios/50/000000/github.png" width="24" height="24"/> GitHub]:::resource
+  PagerDuty[<img src="https://play-lh.googleusercontent.com/E-zhAf4KJ6JDDXmQfQxBprn2sATGYUMkOEqLQX5HAQQtiwDZJg4c8sQd7deb6nCZCwU=w480-h960-rw" width="24" height="24"/> PagerDuty]:::resource
+  KongGateway[<img src="https://raw.githubusercontent.com/Kong/docs.konghq.com/main/app/assets/images/icons/kong-gradient.svg" width="15" height="15"/> Kong Gateway]:::resource
+
+  %% Define the Konnect Service Catalog and its subgraphs
+  subgraph KonnectServiceCatalog[Konnect Service Catalog]
+    direction TB
+
+    %% Define the Billing Service subgraph inside Konnect Service Catalog
+    subgraph BillingService[Billing Service]
+      direction LR
+      KongGatewayService[Kong Gateway service 1]
+      PagerDutyServiceMapping[PagerDuty service 1]
+      GitHubRepoMapping[GitHub repository 1]
+    end
+
+    %% Define the Inventory Service Catalog Service subgraph inside Konnect Service Catalog
+    subgraph InventoryService[Inventory Service]
+      direction LR
+      SomeOtherService[Kong Gateway service 2]
+      AnotherService[PagerDuty service 2]
+      DifferentRepo[GitHub repository 2]
+    end
+
+  end
+
+  %% Connect the resources to the Konnect Service Catalog
+  GitHub -- "GitHub repositories" --> KonnectServiceCatalog
+  PagerDuty -- "PagerDuty services" --> KonnectServiceCatalog
+  KongGateway -- "Kong Gateway services" --> KonnectServiceCatalog
+
+  %% Style the subgraphs
+  class KonnectServiceCatalog konnect
+  class BillingService serviceCatalog
+  class InventoryService serviceCatalog
+{% endmermaid %}
+<!-- vale on-->
+
+> Figure 1: This diagram shows how you can use both external integrations, like GitHub and PagerDuty, as well as built-in integrations like {{site.base_gateway}} to pull resources into Service Catalog. You can then map those resources (like GitHub repositories, PagerDuty services, and {{site.base_gateway}} services) to Service Catalog Services.
+
+## Service Catalog use cases
+
+| You want to... | Then use... |
+| -------------- | ----------- |
+| Keep track of your organization's service ownership by mapping teams to services. | [{{site.konnect_short_name}} Organization Teams](https://cloud.konghq.comorganization/teams) |
```

Comment:
@miko-bautista I'm not sure if the Konnect org teams is the correct feature to use for this use case. Are we adding a new role for Service Catalog? I think I saw on the PagerDuty service, you could add a responsible party. Is there any other way to assign someone to a resource or Service?

Reply:
We still haven't built integrations between Service Catalog and KAuth. However, the idea is that users will be able to associate Konnect Users and/or Konnect Teams to a Service as part of the Service's metadata. 

For now, I've attached a wireframe to give you a visual sense of what I'm talking about. Happy to discuss this in more detail when we meet tomorrow though.

<img width="923" alt="Screenshot 2024-08-17 at 1 08 22 PM" src="https://github.com/user-attachments/assets/0d71572c-a682-41f4-b16d-ff3675dae335">
<img width="922" alt="Screenshot 2024-08-17 at 1 08 29 PM" src="https://github.com/user-attachments/assets/5e34089a-c640-41b9-8b16-71e7a7e1729e">